department-of-veterans-affairs/vets-website
# Update definition list text and bold text to be headers in VAOS

## Description
This PR
- Updates our use of definition lists in facility info to use header tags
- Updates some instances on refresh pages where we were not using header tags at all
- Updates pages in the new appointment flow to fix heading order issues

How to test
- Make sure va_online_scheduling_homepage_refresh flag is off
- Open http://localhost:3001/health-care/schedule-view-va-appointments/appointments/
- Inspect facility information in cards
- Start new appointment flow and view facility info on clinic choice page
- Turn va_online_scheduling_homepage_refresh flag on
- View headings on detail page

## Testing done
Local testing, axe browser scan

## Acceptance criteria
- [ ] Headings are correct and pass axe browser scan

## Definition of done
